2023/political-donations-by-party/main.js

```diff
@@ -0,0 +1,149 @@
+
+    //container to hold the grid
+    var dem_container = svg.append('g').attr('transform', dem_translate);
+
+    dem_container
+        .selectAll('square')
+        .data(demData)
+        .enter()
+        .append('rect')
+        .attr('class', 'square')
+        .attr('id', function (d) {
+            return 'id' + d;
+        })
+        .attr('x', function (d) {
+            return x(d % demNumCols);
+        })
+        .attr('y', function (d) {
+            return y(Math.floor(d / demNumCols));
+        })
+        .attr('height', 15)
+        .attr('width', 15)
+        .attr('fill', dem_square)
+        .style('stroke', dem_square);
+
+    const rep_translate =
+        'translate(' + width * 0.3 + ',' + height * 0.61 + ')';
+
+    var rep_container = svg.append('g').attr('transform', rep_translate);
+
+    rep_container
+        .selectAll('square')
+        .data(repData)
+        .enter()
+        .append('rect')
+        .attr('class', 'square')
+        .attr('id', function (d) {
+            return 'id' + d;
+        })
+        .attr('x', function (d) {
+            return x(d % repNumCols);
+        })
+        .attr('y', function (d) {
+            return y(Math.floor(d / repNumCols));
+        })
+        .attr('height', 15)
+        .attr('width', 15)
+        .attr('fill', rep_square)
+        .style('stroke', rep_square);
+
+    var title = svg
+        .append('text')
+        .attr('class', 'title')
+        .attr('x', 175)
+        .attr('y', 100)
+        .attr('text-anchor', 'start')
+        .attr('font-weight', 700)
+        .attr('font-size', 20)
+        .text('Since 2016, University of Chicago professors have donated...');
+
+    var title = svg
+        .append('text')
+        .attr('class', 'rep-text')
+        .attr('x', 290)
+        .attr('y', 380)
+        .attr('font-weight', 700)
+        .attr('text-anchor', 'middle')
+        .style('fill', rep_square)
+        .text('$63,000')
+        .append('tspan')
+        .attr('font-weight', 500)
+        .style('fill', 'black')
+        .text(' to Republican');
+
+    var title = svg
+        .append('text')
+        .attr('class', 'rep-text')
+        .attr('x', 290)
+        .attr('y', 400)
+        .attr('text-anchor', 'middle')
+        .text('candidates and PACs');
+
+    var title = svg
+        .append('text')
+        .attr('class', 'rep-text')
+        .attr('x', 630)
+        .attr('y', 380)
+        .attr('text-anchor', 'middle')
+        .attr('font-weight', 700)
+        .style('fill', dem_square)
+        .text('$2,000,000')
+        .append('tspan')
+        .attr('font-weight', 500)
+        .style('fill', 'black')
+        .text(' to Democratic');
+
+    var title = svg
+        .append('text')
+        .attr('class', 'rep-text')
+        .attr('x', 630)
+        .attr('y', 400)
+        .attr('text-anchor', 'middle')
+        .text('candidates and PACs');
+};
+
+//call function to draw the graph
+drawGraph();
```

Comment:
If you're just gonna call the function and not append it to some element, it might be better to just use an anonymous function. Something like this:
```javascript
(function() {
    console.log('doing a thing');
})();
```